build.gradle

```diff
@@ -72,9 +72,8 @@ repositories {
 }
 
 dependencies {
-
+	implementation 'uk.gov.laa.ccms.data:data-api:0.0.6-690e561-SNAPSHOT'
```

Comment:
have the data api changes been merged?

Reply:
no idea why I was on that snapshot version. I've rolled it back to 0.0.5.